operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -688,3 +688,77 @@ Expected output will be something like:
     }
 ]
 ```
+
+### Checking the status of remote build nodes
+
+Before a job is set up to be launched on a remote build node, IMS checks that it is
+ready to run remote jobs. If a node is registered as a remote build node, but is not
+able to run remote jobs, that node will not have remote jobs scheduled on it. The
+status of the remote build nodes may be queried. The status object will have a field
+`ableToRunJobs` that contains a boolean indicating if the node is able to run jobs.
+If the node is not able to run jobs, the status object will document the issue with
+running jobs on that node.
+
+* NOTE: a value of '10000' for `numCurrentJobs` indicates that the actual number of
+   current jobs on the node was not able to be determined.
+
+(`ncn-mw#`) To check the status of a particular remote build node:
+
+```bash
+cray ims remote-build-nodes status describe "${IMS_REMOTE_NODE_XNAME}"
+```
+
+If the node is ready to run jobs, the output will look something like:
+
+```text
+{
+  "ableToRunJobs": true,
+  "nodeArch": "x86_64",
+  "numCurrentJobs": 0,
+  "podmanStatus": "Podman present at /usr/bin/podman",
+  "sshStatus": "SSH connection established.",
+  "xname": "x3000c0s3b0n0"
+}
+```
+
+If the node is not able to run jobs, the output will look something like:
```

Comment:
So, all of those are bad values, but any one of the could result in ableToRunJobs being set to 'false' , correct? 
If so, I think you could make that clearer.

Reply:
I am not sure what you mean by 'clearer'. The intent is to provide a reason for the user so they can go fix it. The 'unknown' values are because they can't be queried if we can't ssh into the node. This is a struct so I can't drop the values from the return object.

Reply:
There's a certain amount of irony in my asking for clarity and not providing you with enough information to make the change.

What I meant by clearer is that I thought the documentation should say that _any one of these values_ in the structure below is enough to make the node ineligible to run jobs. When you say that it might look something like this, and then the node has four different things wrong with it, it could convey to the reader that an ineligible node must have all four of those things wrong with it. When, in fact, an ineligible node could actually look like this

{
  "ableToRunJobs": false
  "nodeArch": "unknown",
  "numCurrentJobs": 0,
  "podmanStatus": "Podman present at /usr/bin/podman",
  "sshStatus": "SSH connection established.",
  "xname": "x3000c0s3b0n0"
}

or this

{
  "ableToRunJobs": false,
  "nodeArch": "x86_64",
  "numCurrentJobs": -1,
  "podmanStatus": "Podman present at /usr/bin/podman",
  "sshStatus": "SSH connection established.",
  "xname": "x3000c0s3b0n0"
}
or this
{
  "ableToRunJobs": false,
  "nodeArch": "x86_64",
  "numCurrentJobs": 0,
  "podmanStatus": "Unknown",
  "sshStatus": "SSH connection established.",
  "xname": "x3000c0s3b0n0"
}

or this

{
  "ableToRunJobs": false,
  "nodeArch": "x86_64",
  "numCurrentJobs": 0,
  "podmanStatus": "Podman present at /usr/bin/podman",
  "sshStatus": "Unable to connect to node. Error: [Errno -2] Name does not resolve",
  "xname": "x3000c0s3b0n0"
}

Although I suspect with this last condition, that the podmanStatus would also be unknown as the node could not be reached. Possibly the numCurrentJobs and nodeArch might also be unknown, but I don't know if those are determined by introspecting the node, or if you keep that information separate in a database somewhere.